wrappers/python/pyrs_processing.cpp

```diff
@@ -48,8 +48,8 @@ void init_processing(py::module &m) {
 
     py::class_<rs2::processing_block, rs2::options> processing_block(m, "processing_block", "Define the processing block workflow, inherit this class to "
                                                                      "generate your own processing_block.");
-    processing_block.def(py::init([](std::function<void(rs2::frame, rs2::frame_source&)> processing_function) {
-            return new rs2::processing_block(processing_function);
+    processing_block.def(py::init([](std::function<void(rs2::frame, rs2::frame_source*)> processing_function) {
```

Comment:
Is this still needed?

Reply:
Yes, because frame_source is non-copyable and pybind11 still creates copy of it